### PR TITLE
UI test happy path on multiple browsers

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
No coverage for the happy path - file upload on browsers - we support.

# Solution
Run the happy path - jpg + PDF upload on Edge and Firefox. The file upload is not supported on Safari so we cannot add this browser see this [Github issue](https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/4220).

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [x] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

File: test/config.json [hunk 1]
```diff
@@ -23,9 +23,37 @@
...
+    },
+    {
+      "file": "test/specs/multipleBrowsersDocumentUpload.js",
+      "timeout": 120000,
+      "browsers": [
+        {
+          "browserName": "firefox",
+          "browser_version": "74.0 beta",
+          "os": "OS X",
+          "os_version": "Catalina",
```
Comment (kopijunkie) [inline test/config.json]: Would it be worth adding Firefox on Windows also for the multiple browsers testing while we're at it?

File: test/specs/multipleBrowsersDocumentUpload.js [hunk 2]
```diff
@@ -5,15 +5,15 @@ const options = {
   pageObjects: ['Confirm', 'DocumentSelector', 'Welcome', 'DocumentUpload']
 }
 
-describe('DOCUMENT UPLOAD ON IE11', options, ({driver, pageObjects}) => {
+describe('DOCUMENT AND PDF UPLOAD ON MULTIPLE BROWSERS', options, ({driver, pageObjects}) => {
   const {welcome, documentSelector, confirm, documentUpload} = pageObjects
 
-  it('should upload document on IE11 browser', async () => {
+  it('should upload document with jgp', async () => {
```
Comment (kopijunkie) [inline test/specs/multipleBrowsersDocumentUpload.js]: JPG not "jgp" :)

File: test/specs/multipleBrowsersDocumentUpload.js [hunk 3]
```diff
@@ -5,15 +5,15 @@ const options = {
...
     goToPassportUploadScreen(driver, welcome, documentSelector)
     uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
   })
 
-  it('should upload PDF on IE11 browser', async () => {
+  it('should upload document with pdf', async () => {
```
Comment (kopijunkie) [inline test/specs/multipleBrowsersDocumentUpload.js]: Minor: should be all caps since it's an abbreviation

Comment (bartzet) [reply test/specs/multipleBrowsersDocumentUpload.js]: Done

File: test/specs/multipleBrowsersDocumentUpload.js (hunk 2 above)
Comment (bartzet) [reply test/specs/multipleBrowsersDocumentUpload.js]: Done

File: test/config.json (hunk 1 above)
Comment (bartzet) [reply test/config.json]: Added :)

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

File: test/config.json [hunk 4]
```diff
@@ -23,9 +23,51 @@
...
+        }
+      ]
+    },
+    {
+      "file": "test/specs/multipleBrowsersDocumentUpload.js",
+      "timeout": 120000,
+      "browsers": [
+        {
+          "browserName": "edge",
+          "browser_version": "79.0 beta",
```
Comment (kopijunkie) [inline test/config.json]: Just noticed this, but why `79.0 beta` version for Edge?

Comment (bartzet) [reply test/config.json]: Actually, the latest version of Edge BS supports on Windows 8 is 81 beta so I changed it. They also have 80.0.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

File: test/config.json (hunk 4 above)
Comment (kopijunkie) [reply test/config.json]: I would suggest to go with the latest stable version (80.0) rather than the beta.

Comment (bartzet) [reply test/config.json]: Done :shipit:

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-944-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://944-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**